src/main/java/com/multithread/Application.java

```diff
@@ -0,0 +1,14 @@
+package com.multithread;
+
+public class Application {
+    private static final int ITERATIONS_NUMBER = 100;
+    private static final int COUNTER_START = 0;
+
+    public static void main(String[] args) {
+        Counter counter = new Counter(COUNTER_START);
```

Comment:
```suggestion
        Counter counter = new Counter();
```
0 це дефолтне значення для числових типів

Reply:
done